File: problems/e/electricaloutlets/electricaloutlets.cpp
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <iostream>
#include <string>

using namespace std;

int main() {
    int n, k, o, total_strips;

    cin >> n;


    for (int i=0; i<n; i++) {
        total_strips = 0;        
        cin >> k;
        total_strips -= k - 1;
        
        for (int j=0; j<k; j++) {
            cin >> o;
            total_strips += o;
        }
        
        cout << total_strips << endl;
    }
}

File: problems/e/electricaloutlets/electricaloutlets.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    n = int(input())
    for _ in range(n):
        list_strips = [int(val) for val in input().split()]
        sum_strips = sum(list_strips[1:])
        print(sum_strips - (list_strips[0] - 1))

if __name__ == '__main__':
    main()